Parsers/ASimAuthentication/Parsers/ASimAuthenticationSshd.yaml

```diff
@@ -0,0 +1,105 @@
+Parser:
+  Title: Authentication ASIM parser for Syslog sshd
+  Version: '0.1.1'
+  LastUpdated: November 21, 2022
+Product:
+  Name: sshd
+Normalization:
+  Schema: Authentication
+  Version: '0.1.1'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing Syslog opensshd sign in logs to the ASIM Authentication schema.
+ParserName: ASimAuthenticationSshd
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let SSHDSignInAuthorized=(disabled:bool=false){
+  Syslog | where not(disabled)
+    | where ProcessName == "sshd" and SyslogMessage has 'Accepted '
+    | extend
+    EventVendor = 'OpenBSD'
+    , EventProduct = 'sshd'
+    , EventCount = int(1)
+    , EventSchema = 'Authentication'
+    , EventSchemaVersion = '0.1.1'
+    , EventResult = 'Success'
+    , EventStartTime = TimeGenerated
+    , EventEndTime = TimeGenerated
+    , EventType = 'Logon'
+    , DvcHostname = Computer
+    //, DvcIpAddr = extract(@'\d{1,3}\.\d{1.3}\.\d{1,3}\.\d{1,3}', 1, SyslogMessage)
+    , DvcIpAddr = extract("from (([0-9]{1,3})\\.([0-9]{1,3})\\.([0-9]{1,3})\\.(([0-9]{1,3}))) port",1,SyslogMessage)
```

Comment:
For sshd, the address is SrcIpAddr (and should be aliased to IpAddr and Src as well)